.github/workflows/semver-checks.yml

```diff
@@ -37,3 +37,31 @@ jobs:
           message: |
             :rotating_light: API BREAKING CHANGE DETECTED
 
+  Feature:
+    name: Feature Semver - stable toolchain
```

Comment:
Shouldn't say semver, it should say "Non additive cargo features"

Reply:
Done, thanks!